test/detect/webidl/default.txt

```diff
@@ -0,0 +1,24 @@
+/// Main namespace.
+namespace Namespace {
+    interface interface_identifier { /* interface_members... */ };
```

Comment:
Note that WebIDL namespaces are different to, say, C++ namespaces, in that they don't contain interfaces (only attributes or methods). 

Maybe replace the first namespace declaration with:

```
namespace VectorUtils {
  readonly attribute Vector unit;
  double dotProduct(Vector x, Vector y);
  Vector crossProduct(Vector x, Vector y);
};
```

But the other declarations are fine.